README.md

```diff
@@ -516,3 +520,28 @@ formatArrayToDelimitedList({ arrayToFormat: undefined });
 
 // returns undefined
 ```
+
+<a name="generateAllTypes"></a>
+
+## generateAllTypes({ exclude = '' })
+
+Helper for generating all available JSON schema types.
+You may use the "exclude" parameter to exclude any unwanted type 
+
+**Kind**: global function
+
+| Param         | Type                | Default        | Description                                                                      |
+| ------------- | ------------------- | -------------- | -------------------------------------------------------------------------------- |
+| exclude | <code>String</code>  | <code>''</code> | String of types separated by comma to be excluded. |
+
+**Example**:
+
+```js
+generateAllTypes({})
+
+// returns ['string','number', 'object','array','boolean','null']
+
+generateAllTypes({exclude: 'null, boolean'})
+
+// returns ["string", "number", "object", "array"]
```

Comment:
I'd clean up the example outputs so that strings are represented consistently and spacing is consistent